### ISSUE TITLE
Mistral model problem!

### ISSUE THREAD

**Issue by Dracanto** [camenduru/inferencebot#2]
First, thank you for the project, it's really fantastic!

I'm having trouble using the mistral model. The following message appears immediately after the first chat request:

The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:2 for open-end generation.

Could you please help me resolve it?

Thank you very much
